README.markdown

```diff
@@ -11,21 +11,20 @@ Installation:
 
 Usage:
 
-	upstream backend {                                                                              
-		jdomain www.baidu.com; #port=80                                                             
-		#jdomain www.baidu.com port=8080; #port=8080
```

Comment:
probably include this line updated appropriately, and update the domain used on both lines. Enumerating the supported usages seems like a good idea.